src/ButtonSelect/ButtonSelect.js

```diff
@@ -0,0 +1,46 @@
+
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+
+const menuStyles = {
+  width: '100%',
+  top: '100%'
+};
+
+const ButtonSelect = ({
+  text,
+  children,
+  ...Attr
+}) => {
+  const [toggled, setToggled] = useState(false);
+
+  return (
+    <Fragment>
+      <div className="more-menu__wrapper">
+        <Button onClick={() => setToggled(!toggled)} {...Attr}>
+          <i className="material-icons">{`keyboard_arrow_${toggled ? 'up' : 'down'}`}</i>
```

Comment:
If up and down are modifiers should follow BEM convention with -- before.

Reply:
It is a class of material design, it is external